packages/flutter/lib/src/animation/scheduler.dart

```diff
@@ -2,148 +2,10 @@
-    for (SchedulerCallback callback in localPostFrameCallbacks)
-      invokeCallback(callback, timeStamp);
-
-    _inFrame = false;
-  }
-
-  void invokeCallback(SchedulerCallback callback, Duration timeStamp) {
-    assert(callback != null);
-    try {
-      callback(timeStamp);
-    } catch (exception, stack) {
-      if (debugSchedulerExceptionHandler != null) {
-        debugSchedulerExceptionHandler(exception, stack);
-      } else {
-        print('-- EXCEPTION IN SCHEDULER CALLBACK --');
-        print('$exception');
-        print('Stack trace:');
-        print('$stack');
-      }
-    }
-  }
-
-  /// Call callback every frame.
-  void addPersistentFrameCallback(SchedulerCallback callback) {
-    _persistentCallbacks.add(callback);
-  }
-
-  /// Schedule a callback for the next frame.
-  ///
-  /// The callback will be run prior to flushing the main rendering pipeline.
-  /// Typically, requestAnimationFrame is used to throttle writes into the
-  /// rendering pipeline until the system is ready to accept a new frame. For
-  /// example, if you wanted to tick through an animation, you should use
-  /// requestAnimation frame to determine when to tick the animation. The callback
-  /// is passed a timeStamp that you can use to determine how far along the
-  /// timeline to advance your animation.
-  ///
-  /// Callbacks in invoked in an arbitrary order.
-  ///
-  /// Returns an id that can be used to unschedule this callback.
-  int requestAnimationFrame(SchedulerCallback callback) {
-    _nextCallbackId += 1;
-    _transientCallbacks[_nextCallbackId] = callback;
-    ensureVisualUpdate();
-    return _nextCallbackId;
-  }
-
-  /// Cancel the callback identified by id.
-  void cancelAnimationFrame(int id) {
-    assert(id > 0);
-    _transientCallbacks.remove(id);
-    _removedIds.add(id);
-  }
-
-  /// Schedule a callback for the end of this frame.
-  ///
-  /// If a frame is in progress, the callback will be run just after the main
-  /// rendering pipeline has been flushed. In this case, order is preserved (the
-  /// callbacks are run in registration order).
-  ///
-  /// If no frame is in progress, it will be called at the start of the next
-  /// frame. In this case, the registration order is not preserved. Callbacks
-  /// are called in an arbitrary order.
-  void requestPostFrameCallback(SchedulerCallback callback) {
-    _postFrameCallbacks.add(callback);
-  }
-
-  /// Ensure that a frame will be produced after this function is called.
-  void ensureVisualUpdate() {
-    if (_haveScheduledVisualUpdate)
-      return;
-    ui.window.scheduleFrame();
-    _haveScheduledVisualUpdate = true;
-  }
-}
-
-/// A singleton instance of Scheduler to coordinate all the callbacks.
-final Scheduler scheduler = new Scheduler._();
+export '../scheduler/animation_scheduler.dart' show
+    debugSchedulerExceptionHandler,
+    Scheduler,
+    scheduler,
+    SchedulerCallback,
+    SchedulerExceptionHandler,
+    timeDilation;
```

Comment:
Instead of having this forwarding declaration, we should just update all the references to point to the new location.